scripts/test.py

```diff
@@ -120,18 +120,19 @@ def do():
     coda_exe_path = os.path.join(coda_app_path, 'cli/src/coda.exe')
     coda_exe = os.path.join(coda_build_path, coda_exe_path)
 
-    if os.system('which logproc') == 0:
-        logproc_exe = 'logproc'
-        build_targets = coda_exe
-    else:
-        logproc_exe_path = os.path.join(coda_app_path, 'logproc/logproc.exe')
-        logproc_exe = os.path.join(coda_build_path, logproc_exe_path)
-        build_targets = '%s %s' % (coda_exe_path, logproc_exe_path)
+    with open(os.devnull, 'w') as null:
+        if subprocess.call(['which', 'logproc'], stdout=null, stderr=null) == 0:
+            logproc_exe = 'logproc'
+            build_targets = coda_exe
```

Comment:
this was already present but i notice that the other `build_targets` instead includes `coda_exe_path`. I think dune should be able to use either, so probably not a problem, but I'd prefer if they agreed.

Reply:
Woops. Fixing that now.